spec/integration/create_a_document_api_down_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Create a document when the API is down", type: :feature do
+  scenario "User creates a document without publishing" do
+    given_i_start_to_create_a_document
+    and_the_publishing_api_is_down
+    when_i_submit_the_form
+    then_i_see_the_document_exists
+    and_the_preview_creation_failed
+  end
+
+  def given_i_start_to_create_a_document
+    visit "/"
+    click_on "New document"
+
+    choose "News"
+    click_on "Continue"
+
+    choose "Press release"
+    click_on "Continue"
+  end
+
+  def and_the_publishing_api_is_down
+    @request = stub_publishing_api_put_content(Document.last.content_id,
+                                               hash_including(title: "A great title"))
```

Comment:
Is the linter suggesting you indent like this? doesn't seem very GOV.UK ruby (even though I quite like it) and actually surprised if it's not telling you off.

Reply:
My editor indents like that automatically. I prefer massive indentation over hanging '('s.

Reply:
Yeah - it is more common in ruby to be like that. I thought our rubocop moaned at that but presumably not.